docs/source/intro/install_conda.rst

```diff
@@ -84,6 +84,12 @@ This is the recommended method for users on shared systems and systems where the
        Last name: name
        Institution: where-i-work
 
+   .. admonition:: Having problems auto-detecting the PostgreSQL setup?
```

Comment:
Do we really need a title here? I think the current one also sounds a bit weird since it would be AiiDA having issues with auto-detecting the setup, not the user.